pkgpanda/build/__init__.py

```diff
@@ -37,8 +37,8 @@ def __init__(self):
         self.environment = dict()
         self.container = str()
 
-    def run(self, cmd):
-        docker = ["docker", "run"]
+    def run(self, name, cmd):
+        docker = ["docker", "run", "--name={}".format(name)]
```

Comment:
conflicting names result in non-launching containers, so after the first failure this will fail to ever launch again without manual intervention, could we add a bit of a random name to the end of this to make it so by default run w/ fail -> run just works?